types/keypair/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for keypair 1.0
+// Project: https://www.npmjs.com/package/keypair
```

Comment:
It's better to use its project url(https://github.com/juliangruber/keypair) rather than npm package url

Reply:
+1